inst/rmd/h/default.html

```diff
@@ -613,5 +613,28 @@ <h4 class="date"><em>$date$</em></h4>
 </script>
 $endif$
 
+<!-- apply pandoc div.sourceCode style to pre.sourceCode instead -->
+<script>
+  (function() {
+    var stylesheets = document.styleSheets;
+    for(var i=0; i<stylesheets.length; i++){
+      var rules = stylesheets[i].cssRules;
```

Comment:
Need `try/catch` here since the `cssRules` property is not necessarily accessible: https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet#Notes